math/src/field/fields/fft_friendly/mod.rs

```diff
@@ -1,4 +1,6 @@
 /// Implemenation of the Babybear Prime field p = 2^31 - 2^27 + 1
 pub mod babybear;
+/// Implemenation of the quartic extension of the babybear field
+pub mod quartic_babybear;
```

Comment:
quadratic

Reply:
changed file name to quadratic_babybear.rs and update the mod name.